src/features/customerForm/CustomerEditForm.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+
+import CustomerForm from './CustomerForm';
+import { CustomerFormValues, FormProps } from './types';
+import LoadingSpinner from '../../common/spinner/LoadingSpinner';
+import { CUSTOMER_FORM_QUERY } from './queries';
+import { CUSTOMER_FORM } from './__generated__/CUSTOMER_FORM';
+import { createUpdateInputs, getCustomerFormValues, getIdentifiers } from './utils';
+import { UPDATE_BERTH_SERVICES_PROFILE_MUTATION, UPDATE_PROFILE_MUTATION } from './mutations';
+import { UPDATE_PROFILE, UPDATE_PROFILEVariables as UPDATE_PROFILE_VARS } from './__generated__/UPDATE_PROFILE';
+import {
+  UPDATE_BERTH_SERVICES_PROFILE,
+  UPDATE_BERTH_SERVICES_PROFILEVariables as UPDATE_BERTH_SERVICES_PROFILE_VARS,
+} from './__generated__/UPDATE_BERTH_SERVICES_PROFILE';
+
+export interface CustomerEditFormProps
+  extends Omit<FormProps<CustomerFormValues>, 'initialValues' | 'onCreate' | 'onDelete'> {
+  customerId: string;
+}
+
+const CustomerEditForm = ({ customerId, onCancel, onSubmit, refetchQueries }: CustomerEditFormProps) => {
```

Comment:
Should this be called `CustomerFormContainer`?

Reply:
There will be a `CustomerAddForm` as well, ticket VEN-773

Reply:
I think we should include `Container` to the name somehow since it's a container, what is your suggestion then? :)

Reply:
I guess this could be named `CustomerEditFormContainer`